frontend/javascripts/oxalis/view/right-menu/mapping_info_view.js

```diff
@@ -262,12 +262,28 @@ class MappingInfoView extends React.Component<Props, State> {
       title,
       dataIndex,
     });
+    const showSegmentation64bitWarning =
+      segmentationLayer && segmentationLayer.originalElementClass === "uint64";
+    const maybeWithTooltipWarningTitle = title =>
+      showSegmentation64bitWarning ? (
+        <React.Fragment>
+          {title}{" "}
+          <Tooltip title={message["tracing.uint64_segmentation_warning"]}>
+            <Icon type="info-circle" style={{ color: "rgb(255, 155, 85)" }} />
```

Comment:
Maybe use `warning` as the type instead. That would better fit with the color and the intention of this message to warn the user that the displayed ids may not be correct.

Reply:
Sure, that makes sense :exploding_head: